packages/aws-cdk-lib/aws-codebuild/lib/project.ts

```diff
@@ -2240,9 +2240,9 @@ export class WindowsBuildImage implements IBuildImage {
       errors.push(`Windows images do not support the '${buildEnvironment.computeType}' compute type`);
     }
 
-    if (!buildEnvironment.fleet && this.type === WindowsImageType.SERVER_2022) {
-      errors.push('Windows Server 2022 images must be used with a fleet');
-    }
+    // if (!buildEnvironment.fleet && this.type === WindowsImageType.SERVER_2022) {
+    //   errors.push('Windows Server 2022 images must be used with a fleet');
+    // }
```

Comment:
Can we remove this instead of commenting it out?

Reply:
```suggestion
```